cypress/integration/resources.spec.js

```diff
@@ -30,9 +35,11 @@ describe('resources', () => {
   const assertThatListHasSameItems = () => compareResourceNames({ shouldBeEqual: true });
```

Comment:
Looks like this function is failing consistently. Not sure why since `assertThatListHasDifferentItems` seems to be working fine.